RadarSDK/RadarSettings.h

```diff
@@ -36,6 +36,10 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)setTracking:(BOOL)tracking;
 + (RadarTrackingOptions *_Nullable)trackingOptions;
 + (void)setTrackingOptions:(RadarTrackingOptions *_Nonnull)options;
++ (void)removeTrackingOptions;
++ (RadarTrackingOptions *_Nullable)previousTrackingOptions;
++ (void)setPreviousTrackingOptions:(RadarTrackingOptions *_Nonnull)options;
++ (void)removePreviousTrackingOptions;
```

Comment:
I would prefer to just make `+[RadarSettings setPreviousTrackingOptions:]` take a nullable argument to indicate removal, but this way follows the pattern of the other `trackingOptions` settings.

Reply:
Generally prefer consistency here, internally and with Android. @corypisano